common/maplib/map.mjs

```diff
@@ -263,6 +276,8 @@ class _SingleGameMap {
       cellSet.dynamic = false;
       this.gameMap.cellSets[i] = CellSet.fromObject(cellSet);
     }
+    this.spawnpointCellSet = [];
+    this.spawn_point_list = [];
```

Comment:
1. CamelCaseNamingConvention is commonly used in JavaScript.
2. I think there is no need to add these two class attributes. See the comment of my review.